.circleci/config.yml

```diff
@@ -21,37 +21,31 @@ e2eTest: &e2eTest
       when: always
 
 orbs:
-  architect: giantswarm/architect@0.6.0
+  architect: giantswarm/architect@0.7.0
 
 version: 2.1
 jobs:
   build:
-    machine: true
+    executor: architect/architect
+    working_directory: /go/src/github.com/giantswarm/azure-operator
     steps:
-      - checkout
+      - checkout:
+          path: /go/src/github.com/giantswarm/azure-operator
```

Comment:
Is this still necessary? I think checkout checks out to the working directory by default and they are equal.

Reply:
Removed